src/test/datascience/notebook/ipywidget.vscode.test.ts

```diff
@@ -1,156 +1,19 @@
-        // Execute cell. It should load and render the widget
-        await runCell(cell);
-
-        // Wait till execution count changes and status is success.
-        await waitForExecutionCompletedSuccessfully(cell);
-
-        // Close notebook and open again
-        await closeNotebooks();
-
-        await openNotebook(testWidgetNb);
-        await waitForKernelToGetAutoSelected();
-        cell = vscodeNotebook.activeNotebookEditor?.document.cellAt(0)!;
-
-        // This flag will be resolved when the widget loads
-        const flag = createDeferred<boolean>();
-        flagForWebviewLoad(flag, vscodeNotebook.activeNotebookEditor?.document!);
-
-        // Execute cell. It should load and render the widget
-        await runCell(cell);
-
-        // Wait till execution count changes and status is success.
-        await waitForExecutionCompletedSuccessfully(cell);
-
-        assert.ok(flag.completed, 'Widget did not load successfully on second execution');
-    });
-    test('Can run widget cells that need requireJS (webview-test)', async function () {
-        // Test runs locally but fails on CI, disabling to be fixed in 5265
-        this.skip();
-        await openNotebook(testWidgetNb);
-        await waitForKernelToGetAutoSelected();
-        // 6th cell has code that needs requireJS
-        const cell = vscodeNotebook.activeNotebookEditor?.document.getCells()![6]!;
-
-        // This flag will be resolved when the widget loads
-        const flag = createDeferred<boolean>();
-        flagForWebviewLoad(flag, vscodeNotebook.activeNotebookEditor?.document!);
-
-        // Execute cell. It should load and render the widget
-        await runCell(cell);
-
-        // Wait till execution count changes and status is success.
-        await waitForExecutionCompletedSuccessfully(cell);
-
-        assert.ok(flag.completed, 'Widget did not load successfully during execution');
-    });
-
-    // Resolve a deferred when we see the target uri has an associated webview and the webview
-    // loaded a widget successfully
-    function flagForWebviewLoad(flag: Deferred<boolean>, targetDoc: NotebookDocument) {
-        const commsList = getNotebookCommunications(targetDoc);
-        assert.equal(commsList.length, 1, 'No webviews found in kernel provider');
-        if (Array.isArray(commsList) && commsList.length > 0) {
-            commsList[0].onDidReceiveMessage((e) => {
-                if (e.type === InteractiveWindowMessages.IPyWidgetLoadSuccess) {
-                    flag.resolve(true);
-                }
-            });
+import { sharedIPyWidgetStandardTests } from './ipywidget.vscode.common';
+
+suite('DataScience - VSCode Notebook - Standard IPyWidgets node', function () {
+    // Use the shared code that runs the tests
+    sharedIPyWidgetStandardTests(
+        this,
+        (_s) => {
+            // Don't have any custom tests so don't need the service container
```

Comment:
If we don't have anything, why have these arguments, might as well remove these 3 arguments.

Reply:
I was thinking I needed to add some things to this because test isn't passing for web (capture screen shot) but web version doesn't support screen shots.

I'll remove them if I don't add anything while debugging the web test failure.